index.html

```diff
@@ -60,125 +65,166 @@ <h2 class="header">Tonic</h2>
                         <li>css</li>
                         <li>javascript</li>
                     </ul>
-                    <button class="project-button">See Project</button>
+                    <button type="submit" class="project-button">See Project</button>
+                    </div>
                 </div>
             
-            <div class="work">
-                <div class="work-image">
+            <div class="work reverse">
+                <div class="work-image lost">
                     <img src="images/snap2.png" alt="work-image">
                 </div>
-                <h2 class="header">Multi-Post
-                    Stories</h2>
-                <div class="work-flow">
-                    <div class="canopy">CANOPY</div>
-                    <div class="dot"></div>
-                    <div class="back-end">Back End Dev</div>
-                    <div class="dot"></div>
-                    <div class="year">2015</div>
+                <img class="desktop"  src="images/Portfolio2.png" alt="images">
+                <div class="labels-2">
+                    <h2 class="header">Multi-Post
+                        Stories</h2>
+                    <div class="work-flow work-two">
+                        <div class="canopy canopy-1">CANOPY</div>
+                        <div class="facebook">FACEBOOK</div>
+                        <div class="dot"></div>
+                        <div class="back-end back-1">Back End Dev</div>
+                        <div class="front-end">Full Stack Dev</div>
+                        <div class="dot"></div>
+                        <div class="year">2015</div>
+                    </div>
+                    <p class="paragraph paragraph-1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <p class="desktop-paragraph">
+                        Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                    </p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button type="submit" class="project-button">See Project</button>
```

Comment:
- Please add a hover effect on the card for each project work showcase, so that when hovered it changes the button color as the image below, just like in the Figma design.  (The image below means, that when the card hovered, you notice the button change). 

##### Figma design
![Screenshot from 2022-06-04 08-45-26](https://user-images.githubusercontent.com/58771507/171989984-ed1c7739-4618-4766-bfa7-c88aeeee4323.png)

Reply:
u cant hover the card to change the button color 
well I think we u hover the card its radius changes and the color
also there is a hover effect on the card image

Reply:
u cant hover the card to change the button color
well I think when we hover the card its radius changes and the color
also, there is a hover effect on the card image